generate_task_xml.py

```diff
@@ -0,0 +1,103 @@
+import getpass
+import os
+import subprocess
+from datetime import datetime
+
+# 取得當前使用者的名稱
+current_user = getpass.getuser()
+
+
+# 取得當前使用者的SID
+def get_user_sid(username):
+    try:
+        output = subprocess.check_output(
+            f'wmic useraccount where name="{username}" get sid', shell=True
+        )
+        sid = output.decode().split("\n")[1].strip()
+        return sid
+    except Exception as e:
+        print(f"Error getting SID for user {username}: {e}")
+        return None
+
+
+current_user_sid = get_user_sid(current_user)
+computer_name = os.environ["COMPUTERNAME"]
+
+# Let the user choose between Miniconda and Anaconda
+print("Please choose your Python distribution:")
+print("1. Miniconda")
+print("2. Anaconda")
+choice = input("Enter your choice (1 or 2): ")
+
+if choice == "1":
+    base_path = f"C:\\Users\\{current_user}\\miniconda3"
+elif choice == "2":
+    base_path = f"C:\\Users\\{current_user}\\anaconda3"
+else:
```

Comment:
Not necessarily a blocker, but can we add support for environments that don't use *conda?

Could either just have user paste path of python or maybe we can assume it's on the $PATH and accessible at runtime.